html/controls/operator.js

```diff
@@ -144,6 +145,13 @@ function setClockControls(value) {
 	$("#TeamTime").find("tr.Control").toggleClass("Show", value);
 }
 
+function setReplaceButton(value) {
+	$("#EnableReplaceButton").prop("checked", value);
+	$("label.EnableReplaceButton").toggleClass("ui-state-active", value);
+	$("#ClockUndo").toggleClass("Hidden KeyInactive", value);
```

Comment:
This isn't working, I'm always seeing both buttons.

I'd suggest always showing both.